bot/bot.go

```diff
@@ -190,6 +190,13 @@ func (b *Bot) MentionReply(m *irc.Message, format string, v ...interface{}) {
 	b.Reply(m, format, v...)
 }
 
+// CheckPerm is a convenience function for checking a user's permissions
+func (b *Bot) CheckPerm(m *irc.Message, perm string) bool {
```

Comment:
I totally dig this. Can it be renamed `HasPerm` though so we get consistent naming?